client/web/src/site-admin/WebhookCreateUpdatePage.tsx

```diff
@@ -238,6 +310,22 @@ function supportedExternalServiceKind(kind: ExternalServiceKind): boolean {
     }
 }
 
+function createUpdateWebhookVariables(webhook: Webhook, id?: string): UpdateWebhookVariables {
```

Comment:
Nitpick: it's a bit confusing to have "create" and "update" in this name when the page is also called `CreateUpdate`

```suggestion
function buildUpdateWebhookVariables(webhook: Webhook, id?: string): UpdateWebhookVariables {
```